package.json

```diff
@@ -4,10 +4,13 @@
   "dependencies": {
     "@babel/preset-react": "^7.14.5",
     "@date-io/date-fns": "1.x",
+    "@emotion/react": "^11.4.1",
+    "@emotion/styled": "^11.3.0",
     "@fontsource/roboto": "^4.5.0",
     "@material-ui/core": "^4.12.2",
     "@material-ui/icons": "^4.11.2",
     "@material-ui/pickers": "^3.3.10",
+    "@mui/material": "^5.0.0",
```

Comment:
I think this relates to my above comment, but this is adding material-ui v5 alongside v4. I'm all for updating our dependencies here, but I don't like having two versions of material-ui sticking around without a plan to update.

Can we just use the v4 components for now (https://v4.mui.com/components/alert/#alert) and then have a separate PR to update the material-ui dependencies?

Reply:
Yeah that's a good point - can lead to problems. I removed the v5 version and replaced with v4.

Resolved